htdocs/comm/propal/class/propal.class.php

```diff
@@ -2816,7 +2816,7 @@ public function classifyBilled(User $user, $notrigger = 0, $note = '')
 		}
 
 		if (!$error) {
-			$modelpdf = $conf->global->PROPALE_ADDON_PDF_ODT_CLOSED ? $conf->global->PROPALE_ADDON_PDF_ODT_CLOSED : $this->model_pdf;
+			$modelpdf = isset($conf->global->PROPALE_ADDON_PDF_ODT_CLOSED) ? $conf->global->PROPALE_ADDON_PDF_ODT_CLOSED : $this->model_pdf;
```

Comment:
you can write `$modelpdf = getDolUserString('PROPALE_ADDON_PDF_ODT_CLOSED', $this->model_pdf);`